scanner/image/scanner/konflux.Dockerfile

```diff
@@ -26,14 +26,14 @@ FROM ${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG}
 LABEL \
     com.redhat.component="rhacs-scanner-v4-container" \
     com.redhat.license_terms="https://www.redhat.com/agreements" \
-    description="This image supports image scanning v4 for RHACS" \
-    io.k8s.description="This image supports image scanning for RHACS" \
+    description="This image supports image scanning v4 for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.description="This image supports image scanning for Red Hat Advanced Cluster Security for Kubernetes" \
     io.k8s.display-name="scanner-v4" \
     io.openshift.tags="rhacs,scanner,stackrox" \
```

Comment:
want to update this to `scanner-v4` since you're already updating this file?

Reply:
Thanks, done.